### DISCUSSION TITLE
如何在文章末尾统一添加一张图片？

### DISCUSSION THREAD

**Discussion by georgehao** [wangchucheng/hugo-eureka#106] [Q&A]
有没有相关的方法或者字段

**Comment by wangchucheng** [comment]
目前暂时没有相应方法

**Comment by georgehao** [comment]
统一加版权信息呢？

**Comment by wangchucheng** [comment]
可以通过自定义hugo模板的方式在生成的文件中添加相同的markdown内容。这种方式可能可以满足你的需要


________________________________
发件人: georgehao ***@***.***>
发送时间: Thursday, May 27, 2021 8:03:11 PM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: C. Wang ***@***.***>; Comment ***@***.***>
主题: Re: [wangchucheng/hugo-eureka] 如何在文章末尾统一添加一张图片？ (#106)


统一加版权信息呢？

―
